airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -104,17 +112,22 @@ def handle_call_rate_limit(self, response, params):
                 max_usage = max(max_usage, usage)
```

Comment:
I don't get why getting the maximum between `max_usage` and `usage` here is useful. We could just check afterward that `usage >= self.call_rate_treshold`.